packages/react-server-dom-webpack/npm/writer.browser.server.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./cjs/react-server-dom-webpack-writer.browser.server.production.min.js');
+  module.exports = require('./cjs/react-server-dom-webpack-writer.browser.production.min.server.js');
```

Comment:
as an outsider this is super confusing, `.browser.server`? so which is it?

Reply:
These suffixes are internal filenames so they would be automatically determined by the environment. This is similar how we have `ReactDOMServer` that has a browser build. As a consumer you don't think about those suffixes.